apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,29 +625,31 @@ var ConversationView = {
     // to slide correctly. Bug 1009541
     this.cancelEdit();
 
-    if (Navigation.isCurrentPanel('thread')) {
```

Comment:
> . Come back to the conversation, and scroll up to show the MMS => boom, there is no image displayed ! I'll file a separate bug and this should be quite easy to fix :)

That's funny because I noticed this wrong condition some weeks ago, when was experimenting with view split for the first time, but I wasn't able to reproduce this issue on flame even with huge mms thread, then I just forgot, not sure why it worked for me :)

Anyway that's interesting!